### PR TITLE
add cron to lauch the CodeQL workflow 

### LINKED ISSUE #42: Fix CodeQL scheduling
> I wonder if we should run it also nightly on main.

_Originally posted by @mmanciop in https://github.com/canonical/alertmanager-operator/pull/39#discussion_r722875617_

> I think it might make sense to run this check on any PR, irrespective of the target branch (these are github workflows we do not current use, but still)

_Originally posted by @mmanciop in https://github.com/canonical/alertmanager-operator/pull/39#discussion_r722874856_

### PR DESCRIPTION
**Pull request by SamirPS**
I add the schedule and cron to launch this workflow at 23:00 UTC every day.

Closes #42.

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ on:
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [main]
+  schedule:
+    - cron : "00 23 * * *"
```
Comment (sed-i) [inline .github/workflows/codeql-analysis.yml:15]:
```suggestion
    - cron: "00 23 * * *"
```

**Review by sed-i — COMMENTED** [review]
Thanks @SamirPS!
LG2M!

**Comment by SamirPS** [comment]
It's nothing, i just update the file with your suggestion

**Review by sed-i — APPROVED** [review]

**Comment by SamirPS** [comment]
i or you need to close the pull request ?

**Pull request merged**